jupyter-extension/jupyterlab_pachyderm/tests/conftest.py

```diff
@@ -30,7 +30,18 @@ def pach_config(request, tmp_path) -> Path:
 
 
 @pytest.fixture
-def app(pach_config) -> tornado.web.Application:
+def pach_config_env_var(request) -> str:
```

Comment:
Bit of a nit, but this name seems misleading to me, and the marker name seems a little vague. Could they be something like `pachd_address_env_var`?